app/services/scenes-collections/config.ts

```diff
@@ -301,6 +301,42 @@ export class ScenesCollectionsService extends PersistentStatefulService<IScenesC
   }
 
 
+  fetchSceneCollectionsSchema(): Dictionary<ISceneCollectionSchema> {
+    const schemes: Dictionary<ISceneCollectionSchema> = {};
+    this.state.scenesCollections.forEach(configName => {
+      const data = fs.readFileSync(this.getConfigFilePath(configName)).toString();
```

Comment:
I'm not sure how often streamdeck is going to be calling this function, but this could potentially get a little slow.  We may need to make this async eventually.

Reply:
Ok